Styling/README.md

```diff
@@ -622,6 +624,31 @@ Converts the number from radians to degrees.
 }
 ```
 
+#### clamp
+
+`clamp(value : Number,  min : Number, max : Number) : Number`
+
+Returns value if it is greater than min and less than max. Otherwise, if value is less than min, min is returned, and if value is greater than max, max is returned.
+
+```json
+{
+    "show" : "clamp(${Angle}, 0.0, 90.0) > 45.0"
```

Comment:
I wasn't exactly sure what would, since to me it's a strange function, but my latest does  
```json
{
    "show" : "(clamp(${Angle}, 0.0, 90.0) / 90.0) > 0.5"
}
```